### ISSUE TITLE
make "save" the default behavior

### ISSUE THREAD

**Issue by PotatoesFall** [jonasoreland/runnerup#1183]
Great app, thanks for making it!

I just tried the app and immediately lost my run. Needing to press the save button at the bottom means that unless you scan the entire UI, or if you make a single mistap, your whole run is gone. Which, believe me, is very frustrating.

Unless discard is pressed, the default behavior should be to save the run.

**Comment by Emilv2** [comment]
I just had exactly the same happen to me, what is the reason behind not autosaving?

**Comment by gerhardol** [comment]
The data is there, go to History. You may need to recalculate in the menu.

There is not much that can be done if the activity is killed, there is no "dafault" action.
The UI should be improved to make save button bigger. But the text on the button need to changed to images make this look OK.

**Comment by Emilv2** [comment]
I lost data, it's not in the History tab. I think I clicked on stop without clicking on save, but retrying that and killing the app manually doesn't result in lost data as you say, so I'm not sure what happened.

I'll come back on this if I manage to reproduce my issue.

**Comment by PotatoesFall** [comment]
> The data is there, go to History. You may need to recalculate in the menu.

Nope, that's the first place I looked.

I think the problem could also be fixed by removing the "back" arrow button in the top left, that's where I went wrong.

**Comment by Alteus** [comment]
Same happened. The app is great, I would say this is the only app which covers what I want. But this behaviour resulted losing my exercise data.

**Comment by wookey** [comment]
Agreed. 'Stop' should imply 'save'. There could be a 'discard' button somewhere if you really didn't want to keep it.
